src/Adapter/BaseAdapter.php

```diff
@@ -60,6 +60,9 @@ private function prepareHeaders($headers, $path, $request = null)
         $headers[] = 'x-signature: ' . Signature::generate(
                 $this->options, $path, $randomString, $request
             );
+        if ($this->options->getLanguage() != null) {
```

Comment:
burada isset kullanmak lazım

Reply:
https://www.w3schools.com/pHP/phptryit.asp?filename=tryphp_func_var_isset

Reply:
@sotuzun isset için şu uyarıyı veriyor: 
Replace with '$this->options->getLanguage() !== null'

Tabi eğer böyle yapacaksak ben kontrolü güncellemeliyim el alışkanlığıyla != kalmış.